public/app/features/dashboard/components/LinksSettings/LinkSettingsEdit.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { useState } from 'react';
+import { css } from 'emotion';
+import { CollapsableSection, Button, TagsInput, Select, Field, Input, Switch } from '@grafana/ui';
+import { SelectableValue } from '@grafana/data';
+import { LinkSettingsMode } from '../DashboardSettings/LinksSettings';
+import { DashboardLink, DashboardModel } from '../../state/DashboardModel';
+
+const NEW_LINK = {
```

Comment:
this capitalized variable naming we usually only use sparingly for value constants (numbers & strings)

Reply:
Sorry, old habits. Have camel cased them. 👍